src/changelog-config.ts

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable node/no-process-env, node/no-sync */
+
+import fs from 'fs';
+import path from 'path';
+
+type PackageJson = {
+  autoChangelog: {
+    packageRename: {
+      originalLastestVersion: string;
+      originalTagPrefix: string;
+    };
+  };
+};
+
+/**
+ * Returns the latest version of the original package in case of package renamed.
+ *
+ * @returns The original package latest version.
```

Comment:
`@returns` tag should include return types for all branches.
```suggestion
 * @returns The latest version string of the original package, or null.
```

Reply:
Sure, adding the return types.
```suggestion
 * @returns {string|null} The latest version of the original package, or null.
```

Reply:
There's a `jsdoc/no-types` rule in `@metamask/eslint-config-typescript` that prevents the `{[type]}` syntax from being used in jsdoc `@returns` tags.

Reply:
As this package doesn't use the `@metamask/eslint-config-typescript`, lint didn't complain. But I can see that in core and it's better to be consistent.

Reply:
`auto-changelog` extends `@metamask/eslint-config-typescript` as well. You should be seeing the linter error when you add `{string|null}`. Maybe try restarting the eslint server in vscode?